test/jdk/javax/net/ssl/templates/TLSBase.java

```diff
@@ -67,42 +64,77 @@ abstract public class TLSBase {
 
     TLSBase() {
         String keyFilename =
-                System.getProperty("test.src", "./") + "/" + pathToStores +
-                        "/" + keyStoreFile;
+            System.getProperty("test.src", "./") + "/" + pathToStores +
```

Comment:
It looks you prefer to 4-spaces indent for these cases.
But line 67 and  71 should have the same indent.
This same to line 68 and 71.